keeper/component/forwarder/state.go

```
@@ -0,0 +1,183 @@
+	var iter collections.KeySetIterator[collections.Pair[int32, string]]
+	if protocolID != nil {
+		rng := collections.NewPrefixedPairRange[int32, string](int32(*protocolID))
+		iter, err = f.pausedCrossChains.Iterate(ctx, rng)
+	} else {
+		iter, err = f.pausedCrossChains.Iterate(ctx, nil)
+	}
+
+	if err != nil {
+		return nil, err
+	}
+	defer iter.Close()
```

golangci-lint (2.2.2)

Error return value of `iter.Close` is not checked

(errcheck)